geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -590,7 +596,7 @@ public void testScenario7() throws Exception {
 
       current_fullpath = jtaObj.currRegion.getFullPath();
       assertEquals("failed retrieving current region fullpath after txn commit, fullpath is: "
-          + current_region, "/" + DEFAULT_RGN + "/region1", current_fullpath);
+          + current_region, SEPARATOR + DEFAULT_RGN + "/region1", current_fullpath);
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?